test/test_images/performance/main.go

```diff
@@ -63,6 +64,8 @@ func init() {
 	// role=aggregator
 	flag.StringVar(&listenAddr, "listen-address", ":10000", "Network address the aggregator listens on.")
 	flag.UintVar(&expectRecords, "expect-records", 1, "Number of expected events records before aggregating data.")
+	flag.StringVar(&additionalTags, "additional-tags", "", "Comma separated tags to tag individual"+
```

Comment:
Old name here as well.

Reply:
@antoineco I kept it as a part of the Exit Criteria. WDYT?

Reply:
Ask for forgiveness, not for permission ;)
Nobody will blame you for using descriptive flag/var names.